packages/kbn-telemetry-tools/package.json

```diff
@@ -8,18 +8,12 @@
     "devOnly": true
   },
   "scripts": {
-    "build": "babel src --out-dir target --delete-dir-on-start --extensions .ts --source-maps=inline",
+    "build": "../../node_modules/.bin/babel src --out-dir target --delete-dir-on-start --extensions .ts --source-maps=inline",
```

Comment:
can we use `npx babel` here instead of manually traversing?

Reply:
In some cases npx will not respect the versions declared in the top level package.json so we do feel its safer and simpler to just use the top level .bin scripts. Also it also assures that if someone has a globally installed package the one executed will be de local defined one (which has happen in the past)